src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-linux`
Test-Connection.Error occurred in Describe block
```powershell
Exception calling "GetHostEntry" with "1" argument(s): "Name or service not known"
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1:24 char:9
```